ArchiSteamFarm/CardsFarmer.cs

```diff
@@ -35,10 +34,17 @@
 
 namespace ArchiSteamFarm {
 	internal sealed class CardsFarmer : IDisposable {
+		internal class Game {
+			public uint AppID;
```

Comment:
Consider making ```AppID``` ```readonly``` and use constructor that ensures it's non-0, like in https://github.com/JustArchi/ArchiSteamFarm/blob/master/ArchiSteamFarm/Trading.cs#L46